src/docgen/view/documentation.ts

```diff
@@ -315,19 +315,25 @@ export class Documentation {
    * Lookup a submodule by a submodule name.
```

Comment:
Maybe update the docs to indicate what we now support?

Reply:
Done.